packages/docs/src/routes/qwikcity/routing/index.mdx

```diff
@@ -62,7 +62,7 @@ src/
     │   └── index.tsx         # https://example.com/blog
     ├── post/
     │   └── [...all]/
-    │       └── index.tsx     # https://example.com/post/2020/id/title/2020/12/03
```

Comment:
Example above (line 55) used route example without additional date `(/post/2020/id/title)`, so I removed it from here.